flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/logical/FlinkAggregateJoinTransposeRule.java

```diff
@@ -488,7 +488,6 @@ private Pair<Aggregate, List<RexNode>> toRegularAggregate(Aggregate aggregate) {
                     aggregate.copy(
                             aggregate.getTraitSet(),
                             aggregate.getInput(),
-                            aggregate.indicator,
```

Comment:
`Aggregate.indicator` was deprecated at https://issues.apache.org/jira/browse/CALCITE-2944
Moreover it's always false
https://github.com/apache/calcite/blob/cd5229b9a371e20de207231d55bcbddf1ff39ec2/core/src/main/java/org/apache/calcite/rel/core/Aggregate.java#L111-L112